gaphor/SysML/requirements/connectors.py

```diff
@@ -46,7 +46,7 @@ class DeriveReqtConnect(DirectedRelationshipPropertyPathConnect):
 
 
 @Connector.register(Classified, RefineItem)
-class RefinefyConnect(DirectedRelationshipPropertyPathConnect):
+class RefineConnect(DirectedRelationshipPropertyPathConnect):
```

Comment:
If this was already in 2.4, it should also require a migration.

Reply:
I don't think the Connectors get loaded in the gaphor model files as an element right? I can load a model with a RefineItem that was saved in version 2.4.2 successfully.

Reply:
Yes. For connectors it fine. Sorry. I misread the decorator :see_no_evil: .